mitmproxy/contentviews/image/png.py

```diff
@@ -0,0 +1,289 @@
+# This is a generated file! Please edit source .ksy file and use kaitai-struct-compiler to rebuild
```

Comment:
CI complains about missing test coverage in this file, so shall we move this into mitmproxy/contrib/kaitaistruct?